src/metadata_tool/tools/parsers.clj

```diff
@@ -21,35 +21,42 @@
             [hickory.select                       :as sel]
             [metadata-tool.sources.confluence     :as cfl]
             [metadata-tool.config                 :as cfg]
+            [clojure.pprint                       :as pp]
             [metadata-tool.sources.metadata       :as md]))
 
 (def not-nil? (complement nil?))
 
 (defn parse-string
   [s]
-  (-> s
-      (str/replace "&nbsp;" " ")
-      (str/replace "\u00A0" " ")
-      str/trim))
+  (if (empty? s)
```

Comment:
If `s` is empty, it will pass through the pipeline unchanged.  I don't think this guard clause is necessary.

Reply:
True, but if it's `nil` it will break; you suggest to replace `empty?` with `nil?` ?